### PR TITLE
fix: #2233

### PR DESCRIPTION
**Pull request by ahmedbodi**
## Please verify the following:

- [ ] `yarn test` **jest** tests pass with new tests, if relevant
- [ ] `README.md` has been updated with your changes, if relevant

### CODE REVIEW

File: boilerplate/package.json [hunk 1]
```diff
@@ -32,6 +32,9 @@
     "expo:build:detox": "detox build -c ios.sim.expo",
     "expo:test:detox": "./bin/downloadExpoApp.sh && detox test --configuration ios.sim.expo"
   },
+  "overrides": {
+    "react-error-overlay": "6.0.9"
```
Comment (frankcalise) [inline boilerplate/package.json:36]: Is this change necessary for this fix?

Comment (ahmedbodi) [reply boilerplate/package.json:36]: Yes

**Comment by kateinkim** [comment]
issue : https://github.com/infinitered/ignite/issues/2233

**Comment by joshuayoes** [comment]
I couldn't update the merge conflict, but this change is now in `master`. https://github.com/infinitered/ignite/pull/2252 

Thanks for working with us on this @ahmedbodi!
